include/event_manager.h

```diff
@@ -283,6 +283,26 @@ void event_manager_trace_event_submission(const struct event_header *eh,
  **/
 int event_manager_trace_event_init(void);
 
+/** Allocate event.
+ * The behavior of this function depends on the actual implementation.
+ * The default implementation of this function is same as k_malloc.
+ * It is annotated as weak and can be overloaded by user.
```

Comment:
Actually `overwritten`/`overridden` ?

Reply:
Also in other places and in commit message

Reply:
In commit message:
`This way custom allocation methods or OOM error handling can be provided.`